src/extension/background-script/connectors/nwc.ts

```diff
@@ -205,4 +217,21 @@ class NWCConnector implements Connector {
   }
 }
 
+function mapTLVRecords(
+  tlvRecords: TLVRecord[] | undefined
+): ConnectorTransaction["custom_records"] | undefined {
+  if (tlvRecords) {
```

Comment:
if(!tlvRecords) { return; } 

wrapping the whole function body is not needed then.